services/rsync/incremental.go

```diff
@@ -0,0 +1,63 @@
+package rsync
+
+import (
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/bcc-code/bccm-flows/paths"
+	"github.com/bcc-code/bccm-flows/utils"
+)
+
+type FileInfo struct {
+	Size    int64
+	Name    string
+	ModTime time.Time
+}
+
+func IncrementalCopy(in, out paths.Path, statCallback func(FileInfo)) error {
+	var fileSize int64
+	doCopy := func() (bool, error) {
+		params := []string{
+			"-a",
+			"--progress",
+			"--inplace",
+			"--append",
+			in.Local(),
+			out.Local(),
+		}
```

Comment:
One thing to test if we want with the wav files: When we finish then do a `-a --inplace`.

* On FC we can set up "re-transfer file head when done" for growing files - i.e. it may work to have growing wav files
* This should re-transfer changes only (but I am not sure if the rest of the file is shifted if it will transfer everything)